### PR TITLE
v17: Set NVME driver timeouts

### PR DESCRIPTION
**Pull request by r7vme**
Fixes: https://github.com/giantswarm/giantswarm/issues/4234

### CODE REVIEW

File: service/controller/v17/cloudconfig/master_template.go [hunk 1]
```diff
@@ -274,6 +274,14 @@ func (e *MasterExtension) Units() ([]k8scloudconfig.UnitAsset, error) {
...
 			Command:      "start",
 		},
+		// Set bigger timeouts for NVME driver.
+		// Workaround for https://github.com/coreos/bugs/issues/2484
+		{
+			AssetContent: cloudconfig.NVMESetTimeoutsUnit,
+			Name:         "nvme-set-timeouts.service",
+			Enable:       true,
+			Command:      "start",
+		},
```
Comment (xh3b4sd) [inline service/controller/v17/cloudconfig/master_template.go]: Do we expect this to get rid of again at some point in the future? If yes, lets have a TODO issue so we do not forget about it.

Comment (r7vme) [reply service/controller/v17/cloudconfig/master_template.go]: Yes, we do. Added. Thanks for pushing this!

**Comment by r7vme** [comment]
cc @tuommaki this is a fix for NVME timeouts. Can you also please include it into our relelease?

File: service/controller/v17/version_bundle.go [hunk 2]
```diff
@@ -12,11 +12,16 @@ func VersionBundle() versionbundle.Bundle {
...
+				Component:   "aws-operator",
+				Description: "Set higher timeouts for NVME driver.",
+				Kind:        versionbundle.KindAdded,
+			},
 		},
 		Components: []versionbundle.Component{
 			{
 				Name:    "calico",
-				Version: "3.0.5",
+				Version: "3.2.0",
```
Comment (r7vme) [inline service/controller/v17/version_bundle.go]: Just align with currect state of 3.6.0

File: service/controller/v17/version_bundle.go [hunk 3]
```diff
@@ -28,7 +33,7 @@ func VersionBundle() versionbundle.Bundle {
 			},
 			{
 				Name:    "etcd",
-				Version: "3.3.3",
+				Version: "3.3.8",
```
Comment (r7vme) [inline service/controller/v17/version_bundle.go]: same here

**Pull request merged**